backend/auth.ts

```diff
@@ -72,3 +74,16 @@ export async function validateRequest(
 
   return result;
 }
+
+export async function requireLogin(
+  req: NextApiRequest,
+  res: NextApiResponse,
+): Promise<{ user: LuciaUser; session: Session }> {
+  const userDetails = await validateRequest(req, res);
+  // CHECK THIS, olisiko parempi && eikä ||
+  if (!userDetails.session || !userDetails.user) {
+    throw new HttpError('You are unauthorized!', 401);
+  } else {
+    return userDetails;
+  }
```

Comment:
`else`-statement on turha tässä. Poistan samalla kun poistan `CHECK THIS` -kommentin